profiles/source/Source_4K_Display.yaml

```diff
@@ -1,9 +1,7 @@
 dsDisplay:
   Type: source
   Name: xione
+  Number_of_ports: 0
```

Comment:
There is a problem here for source devices, if source ports are zero and non of the tests runs for source device except init and term. Its better to add 1 port in yaml and type should be HDMI for source devices

Reply:
done